src/modules/facets/renderSelectedFacets.js

```diff
@@ -1,30 +1,35 @@
 export default function() {
-    const selectedFacets = this.getSelectedFacets();
+    const selectedFacetsInfo = this.getSelectedFacets();
     const selectedRanges = this.getSelectedRanges();
-    const k = Object.keys(selectedFacets);
+    const k = Object.keys(selectedFacetsInfo);
     let selectedUi = ``;
     const {
+        selectedFacets,
         facet
     } = this.options;
+    let itemTemplate  = selectedFacets.itemTemplate;
+    if(facet.selectedFacetItemTemplate) {
```

Comment:
the first priority should be given to `selectedFacets.itemTemplate`. If that is not present then use the deprecated `selectedFacetItemTemplate` . The above logic does the opposite.

Reply:
actually its giving priority to selectedFacets.itemTemplate only. still if someone is using deprecated config, it will be overridden.